diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -378,7 +378,15 @@ controller_interface::CallbackReturn DiffDriveController::on_configure(
     }
     else
     {
-      tf_prefix = tf_prefix + "/";
+      // Make sure prefix does not start with '/' and always ends with '/'
+      if (tf_prefix.front() == '/')
+      {
+        tf_prefix.erase(0,1);
+      }
```

Comment:
The line code in the line 375-378 would be obsolete with this right?. Maybe we can remove that part and only leave this one?

Reply:
My first thought was that this cannot be done as in the case of `tf_prefix == "/"` it would first add the '/' and then remove it again. However, if we also switch the two if statements checking `front()` and `back()`, your suggestion should work. I'll make the change.